test/query.spec.js

```diff
@@ -329,7 +329,7 @@ describe('Query', () => {
 
   it('disjoint path values', async () => {
     const peerId = dht.peerId
-    const values = ['v0', 'v1'].map(Buffer.from)
+    const values = ['v0', 'v1'].map((str) => uint8ArrayFromString(str))
```

Comment:
looks like you could pass `uint8ArrayFromString` directly to map, here and in two more cases below.

Reply:
The signature for the function passed to `Array.map` is `function callback( currentValue[, index[, array]])`, so if you just pass `uint8ArrayFromString` it tries to interpret the `index` as a text encoding and fails.